### DISCUSSION TITLE
Muting specific highlight sounds for specific channels

### DISCUSSION THREAD

**Discussion by MPH2210** [Chatterino/chatterino2#2354] [Ideas]
So, right now you can either mute all the notifications for a channel or none.
The problem for me is: My name is max, and in legit every channel people use "max" to shorten "maximum", so I would get pinged every 10 seconds or so.

If you could open up a menu through the three dots on a channel, where you choose which highlights you want to hear from only that channel and which not, would be super cool.

Hope it was understandable what I meant.

**Comment by Mm2PL** [comment]
Accepted as issue #5219.
